storage/innobase/srv/srv0srv.cc

```diff
@@ -1170,7 +1170,15 @@ srv_refresh_innodb_monitor_stats(void)
 {
 	mutex_enter(&srv_innodb_monitor_mutex);
 
-	srv_last_monitor_time = time(NULL);
+	time_t current_time = time(NULL);
+
+	if (difftime(current_time, srv_last_monitor_time) > 60) {
+		/* We referesh InnoDB Monitor values so that averages are
+		printed from at most 60 last seconds */
```

Comment:
You need to unlock mutex here, right?

Reply:
Yes. Sorry once more.

Reply:
Thanks! I will go through this patch once again soon. So far looks good.